src/definitions.d.ts

```diff
@@ -23,6 +23,7 @@ export interface ExtensionActions {
     setShortcut(command: string, shortcut: string);
     toggleURL(url: string);
     markNewsAsRead(ids: string[]);
+    loadConfig(local: boolean);
```

Comment:
Use destructuring (`options: {local: boolean}`) to make it look clear, otherwise `loadConfig(true)` or `loadConfig(false)` doesn't tell what exactly is changed.